person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
   def initialize(age:, name: 'Unknown', parent_permission: true)
+    super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
```

Comment:
Good job so far for the effort in making the requested changes by the previous reviewer, however the issue is still not resolved. I do have errors when I run either the person.rb or the new test.rb file you created as shown below. Kindly check that the methods are correctly linked and the Person class is correctly initialized. 
Kindly ensure that it is running well before resubmiting for approval. You can even add screenshots. :pray: 

![Screenshot from 2023-08-29 11-07-28](https://github.com/zuhebahmed88091/OOP_School_Library/assets/65233860/9f90f9b8-9652-4b8d-bc73-5ee0303aae50)

![Screenshot from 2023-08-29 11-10-00](https://github.com/zuhebahmed88091/OOP_School_Library/assets/65233860/d3c709db-46c0-44fc-b74d-6d08439a38b6)

Reply:
I have added screenshots in PR message

Reply:
<img width="452" alt="testruby" src="https://github.com/zuhebahmed88091/OOP_School_Library/assets/88501552/16f818b3-ef36-4559-8c4e-7ae45999aed3">

Reply:
@jerryowusu  I think you have to add ->
require_relative 'person'
require_relative 'trimmer_decorator'
require_relative 'capitalized_decorator'

for checking the result in person.rb